package-lock.json

```diff
@@ -9,48 +9,48 @@
       "resolved": "https://registry.npmjs.org/@atlaskit/analytics-next/-/analytics-next-1.1.3.tgz",
       "integrity": "sha512-1r9kU2ZbleLCEpJhoPoor/IBbyRue22sutT/+5b8PY43xRgN4HzYT+lWBJRI9fWpG9yJJXLnr9uAjO8LJUKrCg==",
       "requires": {
-        "prop-types": "15.6.0"
+        "prop-types": "^15.5.10"
```

Comment:
@zbettenbuk, it doesn't make sense to have a `package-lock.json` change without a `package.json` change! Even if you wanted to switch `package-lock.json` to a new npm version, then you'd want the change in a commit of its own which clearly explains what you're doing.